github/pkg/adapter/adapter_test.go

```diff
@@ -407,231 +409,108 @@ var testCases = []testCase{
-	}
-
-	h := &fakeHandler{
-		//handler: tc.sink,
-		handler: sinkAccepted, // No tests expect the sink to do anything interesting
+func (tc *testCase) validateAcceptedPayload(t *testing.T, ce *adaptertest.TestCloudEventsClient) error {
```

Comment:
This function is a test helper, so it should start with `t.Helper()` so that the lines where error occur are correctly reported in logs.

Reply:
Done.